clang/include/clang/Basic/DiagnosticFrontendKinds.td

```diff
@@ -179,7 +179,7 @@ def err_verify_invalid_no_diags : Error<
     "%select{expected|'expected-no-diagnostics'}0 directive cannot follow "
     "%select{'expected-no-diagnostics' directive|other expected directives}0">;
 def err_verify_no_directives : Error<
-    "no expected directives found: consider use of 'expected-no-diagnostics'">;
+    "no expected directives found: consider use of '%0-no-diagnostics'">;
```

Comment:
```suggestion
    "no %0 directives found: consider use of '%0-no-diagnostics'">;
```
no?

Reply:
I guess elsewhere we refer to them as expected diagnostics, it's only in single quotes that we intend it to match the source?

Reply:
this is just some formatting issue. I was following the directions of the original issue, where @tbaederr mentioned changing only the latter expected part of the string.

Reply:
> I guess elsewhere we refer to them as expected diagnostics, it's only in single quotes that we intend it to match the source?

Correct, there are expected (English sense of the word) directives, and there are 'expected' (syntax sense of the word) directives. We want single quotes around the syntax part.

Reply:
Considering this as resolved.